CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_hubspot v0.17.0
+[PR #137](https://github.com/fivetran/dbt_hubspot/pull/137) includes the following updates:
+
+## 🚨 Breaking Changes: Variable Bug Fixes 🚨 
```

Comment:
i agree that this is a breaking change, but is it only breaking change specifically for people who have `hubspot_email_event_sent_enabled` set to False? since it's `true` by default 

if so, perhaps we should call that out